WalletWasabi.Tests/AcceptanceTests/HwiKatas.cs

```diff
@@ -456,4 +457,89 @@ public async Task LedgerNanoXKataAsync()
 		var checkResult = signedTx.Check();
 		Assert.Equal(TransactionCheckResult.Success, checkResult);
 	}
+
+	[Fact]
+	public async Task JadeKataAsync()
+	{
+		// --- USER INTERACTIONS ---
+		//
+		// Connect and initialize your Jade with the following seed phrase:
+		// more maid moon upgrade layer alter marine screen benefit way cover alcohol
+		// Run this test.
+		// displayaddress request(derivation path): approve
```

Comment:
These instructions not corresponds to the ones those are displayed on the device.

Reply:
Agree.